policy-utils/src/parsers.rs

```diff
@@ -67,7 +67,13 @@ pub fn parse_renamable_paths(
 /// mistakes.
 #[cfg(feature = "std")]
 pub fn enforce_leading_backslash(path: &str) -> Cow<str> {
-    if !path.starts_with('/') {
+    let is_special_file = match path {
+        "stdin" => true,
+        "stderr" => true,
```

Comment:
Also, I wonder whether these "special file" names should be constants, rather than matching on string literals (and potentially introducing typos)?  e.g.

```rust
let is_special_file = path == STDIN_FILE_PATH || path == STDOUT_FILE_PATH || path == STDERR_FILE_PATH;
```